composer/utils/object_store/mlflow_object_store.py

```diff
@@ -0,0 +1,392 @@
+        environment variables with the right credentials.
+
+        Unlike other object stores, the DBFS URI scheme for MLFlow artifacts has no bucket, and the path is prefixed
+        with the artifacts root directory for a given experiment/run,
+        `databricks/mlflow-tracking/<experiment_id>/<run_id>/`. However, object names are also sometimes passed by
+        upstream code as artifact paths relative to this root, rather than the full path. To keep upstream code simple,
+        :class:`MLFlowObjectStore` accepts both relative MLFlow artifact paths and absolute DBFS paths as object names.
+        If an object name takes the form of
+        `databricks/mlflow-tracking/<experiment_id>/<run_id>/artifacts/<artifact_path>`,
+        it is assumed to be an absolute DBFS path, and the `<artifact_path>` is used when uploading objects to MLFlow.
+        Otherwise, the object name is assumed to be a relative MLFlow artifact path, and the full provided name will be
+        used as the artifact path when uploading to MLFlow.
+
+    Args:
+        path (str): A DBFS path of the form
+            `databricks/mlflow-tracking/<experiment_id>/<run_id>/artifacts/<path>`.
+            `experiment_id` and `run_id` can be set as the format string placeholders `'EXPERIMENT_ID'` and `'RUN_ID'`.
+
+            If both `experiment_id` and `run_id` are set as placeholders, the MLFlowObjectStore will be associated with
+            the currently active MLFlow run if one exists. If no active run exists, a new run will be created under a
+            default experiment name, or the experiment name specified by the `MLFLOW_EXPERIMENT_NAME` environment
+            variable if one is set.
+
+            If `experiment_id` is provided and `run_id` is not, the MLFlowObjectStore will create a new run in the
+            provided experiment.
+
+            Providing a `run_id` without an `experiment_id` will raise an error.
+        multipart_upload_chunk_size(int, optional): The maximum size of single chunk in an MLFlow multipart upload.
+            The maximum number of chunks supported by MLFlow is 10,000, so the max file size that can
+            be uploaded is `10 000 * multipart_upload_chunk_size`. Defaults to 100MB for a max upload size of 1TB.
+    """
+
+    def __init__(self, path: str, multipart_upload_chunk_size: int = 100 * 1024 * 1024) -> None:
+        try:
+            import mlflow
+            from mlflow import MlflowClient
+        except ImportError as e:
+            raise MissingConditionalImportError('mlflow', conda_package='mlflow>=2.9.2,<3.0') from e
+
+        try:
+            from databricks.sdk import WorkspaceClient
+        except ImportError as e:
+            raise MissingConditionalImportError('databricks', conda_package='databricks-sdk>=0.15.0,<1.0') from e
+
+        tracking_uri = os.getenv('MLFLOW_TRACKING_URI', MLFLOW_DATABRICKS_TRACKING_URI)
```

Comment:
only reason being I didn't realize I could. thanks for pointing it out!

Reply:
I take it back - `mlflow.get_tracking_uri()` falls back to a local mlflow path in the env var isn't set, and it also makes testing a pain because the `tracking_uri` gets set globally by other tests for `MLFlowLogger`, and there's no easy way to just "unset" the active tracking uri. Going to revert back to just reading the env var and defaulting to Databricks directly